### DISCUSSION TITLE
Delayed Responses in STT Streaming

### DISCUSSION THREAD

**Discussion by GauravLabhane** [deepgram/community#860] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - STT Streaming

### Details

We are getting  too much delay in almost all of our requests today. 

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

/v1/listen?punctuate=false&model=nova-2&sample_rate=16000&channels=1&encoding=linear16&language=hi&endpointing=110

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

42990d30-2d5f-4b09-9f44-d98436f79522

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by SandraRodgers** [comment, marked as answer]
Is this something that you are just experiencing today? Can you tell me if the problem still continues? If it's a temporary issue, it's most likely related to your network. We have not had any service issues with STT streaming. You can view the status updates here: https://status.deepgram.com/
